cluster/gce/config-default.sh

```diff
@@ -521,3 +521,6 @@ export GCE_PRIVATE_CLUSTER_PORTS_PER_VM="${KUBE_GCE_PRIVATE_CLUSTER_PORTS_PER_VM
 # Optional: Create apiserver konnectivity server and agent.
 export ENABLE_EGRESS_VIA_KONNECTIVITY_SERVICE="${KUBE_ENABLE_EGRESS_VIA_KONNECTIVITY_SERVICE:-false}"
 export KONNECTIVITY_SERVICE_PROXY_PROTOCOL_MODE="${KUBE_KONNECTIVITY_SERVICE_PROXY_PROTOCOL_MODE:-grpc}"
+
+# Optoinal: Enable Windows CSI-Proxy
```

Comment:
typo: optional

Reply:
en, my keyboard has issues these day. thanks for checking those bad typoos.... :(

Reply:
fixed